crates/engine/tree/src/tree/mod.rs

```diff
@@ -568,6 +568,13 @@ where
             // Check if persistence has completed
             match rx.try_recv() {
                 Ok(last_persisted_block_hash) => {
+                    let Some(last_persisted_block_hash) = last_persisted_block_hash else {
+                        // if this happened, then we persisted no blocks. Let's just `warn` here,
+                        // since this should not happen frequently, and we have multiple checks to
+                        // prevent against sending an empty range to the persistence task
```

Comment:
can we rephrase this slightly and remove the "let's just warn here part"?

> if this happened, then we persisted no blocks because we sent an empty vec of blocks